source/nodejs/adaptivecards-site/pages/_posts/2022/Community-Call-February.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 📣 Community Call - February, 2022
+subtitle: Adaptive Cards Video Player Bot on Teams
+date: 2022-02-10
+featured_image: https://techcommunity.microsoft.com/t5/image/serverpage/image-id/347408iAF616545D5ED026B/image-size/large?v=v2&px=999
+github_username: JeanRoca
+twitter: jpthepm
+---
+
+The following topics are covered in this month’s community call. Demo: **Adaptive Cards Video Player Bot on Teams** (set up an Adaptive Card that plays video content within Teams environment, complete sample code) - [Alexis Kinzelin](https://www.linkedin.com/in/alexis-kinzelin-214b6b20/) (Microsoft), and **Q&A** – [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) and [Alexis Kinzelin](https://www.linkedin.com/in/alexis-kinzelin-214b6b20/) (Microsoft) field general and demo related questions. This call was hosted by [J.P. Roca](http://twitter.com/jpthepm) (Microsoft) | @jpthepm. Recorded on February 10, 2022.
+
+### Watch the call here
+
+<iframe width="680" height="383" src="https://www.youtube.com/embed/-Y7_nNwC20w" title="Community Call February 2022 video" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
![image](https://user-images.githubusercontent.com/21356912/160211222-d94d2843-9a0d-4db5-a5f4-5b48a6c6f9cd.png)
this is what it looks like, looks broken?:)

Reply:
@beervoley I don't think the video will render in markdown since it is an embed. If you go to the specific test site for this PR you should be able to see them:
https://adaptivecardsci.z5.web.core.windows.net/pr/7163/blog/2022/Community-Call-February/